internal/validators/intBetweenFromString.go

```diff
@@ -0,0 +1,56 @@
+package validators
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/helpers/validatordiag"
+)
+
+var _ validator.String = inBetweenFromStringValidator{}
+
+type inBetweenFromStringValidator struct {
+	min, max int32
```

Comment:
Is there a reason to use int32 over int?

Reply:
no specific reason, will change that